Notification Plugin docs (#22)
breez/breez-sdk-liquid-docs: src/notifications/custom_messages.md

# Customising push messages

The Notification Plugin by default handles a specific format of push notification message data sent by the NDS:
```json
{
    "notification_type": "swap_updated",
    "notification_payload": "{ \"id\": \"\", \"status\": \"\" }",
    "app_data": ""
}
```

> If you want to customise the structure or field names of the push notification message data handled by the Notification Plugin, you can follow the code example below for adding these to iOS and Android. In these examples we replace the `notification_type` and `notification_payload` fields with `custom_type` and `custom_payload`.

Reviewer: This explanation is a bit wordy? It's better to keep it brief and let the examples speak for themselves imo.
```suggestion
To customize the push notification message data handled by the Notification Plugin, follow the code examples below for iOS and Android. In these examples, the `notification_type` and `notification_payload` fields are replaced with `custom_type` and `custom_payload`.
```
